Code/Framework/AzCore/AzCore/IO/Streamer/Streamer.h

```diff
@@ -171,6 +174,11 @@ namespace AZ::IO
         //! Call to collect statistics from all the components that make up Streamer.
         void CollectStatistics(AZStd::vector<Statistic>& statistics) override;
 
+        //! Tells AZ::IO::Streamer the report the information for the report to the output.
```

Comment:
typo: the report -> to collect
```suggestion
        //! Tells AZ::IO::Streamer to collect the information for the report to the output.
```

Reply:
Reworded in the next commit.